.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_CONNECTION_STRING=postgresql://postgres:postgres@localhost/jobsgowhere?sslmode=disabled
+AUTH0_DOMAIN=jobsgowhere.au.auth0.com
+AUTH0_CLIENT_ID=rc3Waw3xebsUoG2uE7I2Zb1T5jgSfrTF
```

Comment:
let's move this to environment variables?

Reply:
in go, these are env vars. later we can put it in ci/cd.

Reply:
we should add this to `.gitignore` right?

Reply:
Are we targeting more white collar population? Or general population? I feel that there's a better chance of having gmail and using google account